docs/how-to/add-model-and-api-endpoints.md

```diff
@@ -121,80 +271,216 @@ This is code that serializes objects into strings for the API endpoints, and des
+   1. `basename` is the name used for generating the endpoint names, such as [basename]-list, [basename]-detail, etc. It's in the singular form. This is automatically generated if the viewset definition contains a `queryset` attribute, but it's required if the viewset overrides that with the `get_queryset` function
+      - `reverse("recurring-event-list")` would return `http://localhost/api/v1/recuring-events/`
 
 ### Add API tests
 
-For the CRUD operations, since we're using `ModelViewSet` where all the actions are provided by `rest_framework` and well-tested, it's not necessary to have test cases for them. But here's an example of one
+For the CRUD operations, since we're using `ModelViewSet` where all the actions are provided by `rest_framework` and well-tested, it's not necessary to have test cases for them. But here's an example of one.
 
 1. Import API URL
 
-   https://github.com/fyliu/peopledepot/blob/acd8898e7b0364913cc8ae3f9973dfd846adedcc/app/core/tests/test_api.py#L11
+   ```python
+   RECURRING_EVENTS_URL = reverse("recurring-event-list")
+   ```
+
+   [link to code](https://github.com/hackforla/peopledepot/blob/09e2856b6dd8038aedbbc9b42c3a44009be1fd2f/app/core/tests/test_api.py#L11)
 
 1. Add test case
-   https://github.com/fyliu/peopledepot/blob/acd8898e7b0364913cc8ae3f9973dfd846adedcc/app/core/tests/test_api.py#L70-L81
 
-   1. Use `auth_client` instead of `admin_client`
+   ```python
+   def test_create_recurring_event(auth_client, project):
+       """Test that we can create a recurring event"""
+
+       payload = {
+           "name": "Test Weekly team meeting",
+           "start_time": "18:00:00",
+           "duration_in_min": 60,
+           "video_conference_url": "https://zoom.com/link",
+           "additional_info": "Test description",
+           "project": project.uuid,
+       }
+       res = auth_client.post(RECURRING_EVENTS_URL, payload)
+       assert res.status_code == status.HTTP_201_CREATED
+       assert res.data["name"] == payload["name"]
+   ```
+
+   [link to code](https://github.com/hackforla/peopledepot/blob/097f8f254534c1e53bc23f14ef71afbed0b70fa0/app/core/tests/test_api.py#L150-L163)
+
    1. Given
       1. Pass in the necessary fixtures
       1. Construct the payload
    1. When
       1. Create the object
    1. Then
       1. Check that it's created via [status code](https://www.django-rest-framework.org/api-guide/status-codes/#client-error-4xx)
-      1. Maybe also check the data
+      1. Maybe also check the data. A real test should check all the data, but we're kind of relying on django to have already tested this.
+
+1. Run the test script to show it passing
+
+   ```bash
+   ./scripts/test.sh
+   ```
+
+### Check point 3
+
+This is a good place to pause, check, and save. Refer to the [contributing doc](https://github.com/hackforla/peopledepot/blob/main/docs/CONTRIBUTING.md) sections 3.4-3.9 for details.
```

Comment:
Unfortunately contributing doc was changed so 3.4-3.9 are not there anymore

Reply:
Hmm... maybe it's not a good idea to be referring to contributing. There's a couple more places with this instruction. I'm just going to remove it since the instructions are in the text anyway.